### PR TITLE
pacman: add pt_PT translation

### PR DESCRIPTION
**Pull request by simao-ferreira**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
6.0.1

### CODE REVIEW

File: pages.pt_PT/linux/pacman.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# pacman
+
+> Utilitário para gerir pacotes Arch Linux
+> Sub comandos como `pacman sync` tem a sua própria documentação
+> Mais informações em: <https://man.archlinux.org/man/pacman.8>.
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:5]:
```suggestion
> Mais informações: <https://man.archlinux.org/man/pacman.8>.
```

File: pages.pt_PT/linux/pacman.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# pacman
+
+> Utilitário para gerir pacotes Arch Linux
+> Sub comandos como `pacman sync` tem a sua própria documentação
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:4]:
```suggestion
> Sub comandos como `pacman sync` tem a sua própria documentação.
```

File: pages.pt_PT/linux/pacman.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# pacman
+
+> Utilitário para gerir pacotes Arch Linux
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:3]:
```suggestion
> Utilitário para gerir pacotes Arch Linux.
```

File: pages.pt_PT/linux/pacman.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+> Sub comandos como `pacman sync` tem a sua própria documentação
+> Mais informações em: <https://man.archlinux.org/man/pacman.8>.
+
+- Sincronizar e actualizar todos os pacotes:
+`sudo pacman --sync --refresh --sysupgrade`
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:8]:
```suggestion

`sudo pacman --sync --refresh --sysupgrade`
```

File: pages.pt_PT/linux/pacman.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# pacman
+
+> Utilitário para gerir pacotes Arch Linux
+> Sub comandos como `pacman sync` tem a sua própria documentação
+> Mais informações em: <https://man.archlinux.org/man/pacman.8>.
+
+- Sincronizar e actualizar todos os pacotes:
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:7]: Use this mood please:
```suggestion
- Sincroniza e actualiza todos os pacotes:
```

File: pages.pt_PT/linux/pacman.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+`sudo pacman --sync --refresh --sysupgrade`
+
+- Instalar um novo pacote:
+
+`sudo pacman --sync {{package_name}}`
+
+- Remover um pacote e todas as dependencias:
+
+`sudo pacman --remove --recursive {{package_name}}`
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:16]: Plase translate arguments
```suggestion
`sudo pacman --remove --recursive {{nome_do_pacote}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.pt_PT/linux/pacman.md (hunk 5 above)
Comment (simao-ferreira) [reply pages.pt_PT/linux/pacman.md:7]: I would keep them in infinitive form as the remaining descriptions.

File: pages.pt_PT/linux/pacman.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# pacman
+
+> Utilitário para gerir pacotes Arch Linux.
+> Sub comandos como `pacman sync` tem a sua própria documentação.
+> Mais informações: <https://man.archlinux.org/man/pacman.8>.
+
+- Sincroniza e actualiza todos os pacotes:
```
Comment (navarroaxel) [inline pages.pt_PT/linux/pacman.md:7]: Use the same mood in all the examples.

Comment (simao-ferreira) [reply pages.pt_PT/linux/pacman.md]: Done. Thanks

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**